lib/crawler.js

```diff
@@ -440,81 +505,18 @@ Crawler.prototype.discoverResources = function(resourceData,queueItem) {
 	if (!queueItem.protocol)
 		queueItem.protocol = "http";
 
-	if (!crawler.parseHTMLComments) {
+	if (!crawler.parseHTMLComments)
```

Comment:
Done! I agree, adding some JSCS checking would be cool - good to have an enforced style